.eslintrc.json

```diff
@@ -21,13 +21,20 @@
   },
   "plugins": ["react", "prettier"],
   "rules": {
-    "prettier/prettier": "error",
     "newlines-between": "always",
     "react/forbid-prop-types": [2,
       {
         "forbid": ["object", "array"], "checkContextTypes": false, "checkChildContextTypes": false
       }
     ],
+    "no-unused-vars": [
```

Comment:
Hey, @espoirMur why did you ignore the un-used vars and args with `_` name?
If there is a technical concern which made you ignore it,  please state it. And sometimes, consider communicating such things in through either the team channels or PR description.

Reply:
Thanks, Pacifique for pointing that out.
sometimes you need an unused variable in a function,  and to bypass the rule you need to start it with _.
A use case was in  componentwillReceiveProps, the fucntion takes  nextprops parameter which I don't use at all in the function.